packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -146,21 +146,34 @@ async function updateMasterBranchProtection(repo: Repo, context: Context) {
       checks = customConfig.requiredStatusChecks;
     }
   }
-  await context.github.repos.updateBranchProtection({
-    branch: 'master',
-    owner,
-    repo: name,
-    required_pull_request_reviews: {
-      dismiss_stale_reviews: false,
-      require_code_owner_reviews: false,
-    },
-    required_status_checks: {
-      contexts: checks,
-      strict: config.requireUpToDateBranch,
-    },
-    enforce_admins: true,
-    restrictions: null!,
-  });
+  try {
+    await context.github.repos.updateBranchProtection({
+      branch: 'master',
+      owner,
+      repo: name,
+      required_pull_request_reviews: {
+        dismiss_stale_reviews: false,
+        require_code_owner_reviews: false,
+      },
+      required_status_checks: {
+        contexts: checks,
+        strict: config.requireUpToDateBranch,
+      },
+      enforce_admins: true,
+      restrictions: null!,
+    });
+  } catch (err) {
+    if (err.status === 401) {
+      console.warn(
```

Comment:
Why is a 401 ok in this case?  Any time we do this, adding a meaty comment explaining the situation would be lovely.